influence_benchmark/RL/expert_iteration.py

```diff
@@ -19,11 +19,16 @@ def _format_and_save_trajectories_for_sft(self, selected_trajectories, trajector
         for trajectory in selected_trajectories:
             system_prompt = trajectory["agent_system_prompt"][0]["content"]
             messages = [{"role": "system", "content": system_prompt}]
+
             for msg in trajectory["history"]:
                 if msg["role"] == "agent":
                     messages.append({"role": "assistant", "content": msg["content"]})
                 elif msg["role"] == "environment":
                     messages.append({"role": "user", "content": msg["content"]})
+                elif msg["role"] == "tool_use":
+                    messages.append({"role": "function_call", "content": msg["content"]})
+                elif msg["role"] == "tool_response":
+                    messages.append({"role": "ipython", "content": msg["content"]})
```

Comment:
Seems like this part of the code could be extracted in it's own method in BaseIteration as it's in common with expert iteration